cartography/intel/gcp/sql.py

```diff
@@ -248,6 +305,58 @@ def _load_sql_users_tx(tx: neo4j.Transaction, sql_users: List[Dict], project_id:
     )
 
 
+@timeit
+def load_sql_databases(session: neo4j.Session, data_list: List[Dict], project_id: str, update_tag: int) -> None:
+    session.write_transaction(_load_sql_databases_tx, data_list, project_id, update_tag)
+
+
+@timeit
+def _load_sql_databases_tx(tx: neo4j.Transaction, sql_databases: List[Dict], project_id: str, gcp_update_tag: int) -> None:
+    """
+        :type neo4j_transaction: Neo4j transaction object
+        :param neo4j transaction: The Neo4j transaction object
+
+        :type sql_databases: List
+        :param sql_databases: A list of SQL database
+
+        :type project_id: str
+        :param project_id: Current Google Project Id
+
+        :type gcp_update_tag: timestamp
+        :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+    """
+    ingest_sql_databases = """
+    UNWIND $sql_databases as database
+    MERGE (d:GCPSQLDatabase{id:database.id})
+    ON CREATE SET
+        d.firstseen = timestamp()
+    SET
+        d.name = database.name,
```

Comment:
can you check if there's a status field for the database? if not, let's use the instance state as state for the database since database always resides under instance.

Reply:
state attribute not in response .we use instance state

Reply:
i don't see state attribute in this mapping.